docs/tutorial.js.html

```diff
@@ -267,10 +272,10 @@ <h1 class="page-title">Source: tutorial.js</h1>
     let nextStep;
     if (step) {
       if (typeof step === 'number') {
-        if (step &lt; 0 || step >= this.steps.length) {
+        if (step &lt; 1 || step > this.steps.length) {
```

Comment:
@th11 @stephaniewei Any idea if this would break our usage of chariot in product?